File: CS/dxSampleGroupSort/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraReports.UI;
using ReportSortHelpers;

namespace dxSampleGroupSort {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }
        private void ShowReport() {
            ReportColumnSortHelper sortHelper = new ReportColumnSortHelper(new XtraReport1());
            sortHelper.ShowPreviewDialog();
        }
        private void simpleButton1_Click(object sender, EventArgs e) {
            ShowReport();
        }
    }
}

File: VB/dxSampleGroupSort/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraReports.UI

Namespace dxSampleGroupSort
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Private Sub ShowReport()

            CType(New XtraReport1(), XtraReport1).ShowPreview()
        End Sub
        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
            ShowReport()
        End Sub
    End Class
End Namespace
